### ISSUE TITLE
Need to change the RocketChat Service Title with Alert Status

### ISSUE THREAD

**Issue by ferdousulhaque** [ObjectifLibre/cabot-alert-rocketchat#2]
I have already configured Rocketchat plugin in the cabot application. Thank you for the plugin contribution, really grateful. However, need some tweaking while showing the notification on mobile platforms. When the notification comes it only shows the service name. It would be AWESOME if the alert notification also added with service status small part.

I have found the place to change the config in the code, but I have installed using pip. How to change that config code, I have no idea. Will be helpful, if you can give me a pathway.

```
service_status_template = "{% if service.overall_status ==
service.PASSING_STATUS %}Back to normal{% else %}Reporting {{
service.overall_status }} status{% endif %}: [Link]({{ scheme }}://{{ host
}}{% url 'service' pk=service.id %})" check_error_template = "{% if
check.check_category == 'Jenkins check' %}{% if check.last_result.error %} {{
check.name }} ({{ check.last_result.error|safe }}) {{jenkins_api}}job/{{
check.name }}/{{ check.last_result.job_number }}/console{% else %} {{
check.name }} {{jenkins_api}}/job/{{ check.name
}}/{{check.last_result.job_number}}/console {% endif %}{% else %} {{
check.name }} {% if check.last_result.error %} ({{
check.last_result.error|safe }}){% endif %}{% endif %}" alert_template = "{%
for alias in users %} @{{ alias }}{% endfor %}"
```

**Comment by fydrah** [comment]
Hi @ferdousulhaque,
Could you give me an example of what you actually have vs. what you want ?

I think we could change the code to pass a part of the template (or maybe the whole template) as a variable. I will think about it and give you updates later.
R.

**Comment by fydrah** [comment]
Hi @ferdousulhaque,

Any news about this issue ?

**Comment by ferdousulhaque** [comment]
On mobile notification part on Rocket.chat, we used to get only alert, to know the type like error or recovery, we need to tap and login to the mobile app to get the actual alert reason. By changing in the django file in service_status_template I added status string to the alert title. After that when we get alert, we get a type like error or normal in the title of the notification. Will be helpful, if you can add a title for notification that includes 'is down' and when recovered 'is recovered'.

**Comment by fydrah** [comment]
I just released 0.1.2 version on pypi --> https://pypi.org/project/cabot-alert-rocketchat/0.1.2/
